install-dev/data/db_structure.sql

```diff
@@ -1270,6 +1270,7 @@ CREATE TABLE `PREFIX_orders` (
   KEY (`current_state`),
   KEY `id_shop` (`id_shop`),
   INDEX `date_add`(`date_add`)
```

Comment:
Hi @AdrienPlx it's a nice idea but I think you forgort to modify above line to add the missing comma